app/controllers/oidc_provider/user_infos_controller.rb

```diff
@@ -3,7 +3,8 @@ class UserInfosController < ApplicationController
     before_action :require_access_token
 
     def show
-      render json: AccountToUserInfo.new.(current_token.authorization.account, current_token.authorization.scopes)
+
+      render json: AccountToUserInfo.new.(current_token.authorization.account, current_token.authorization.scopes, current_token.authorization.nonce)
```

Comment:
We don't need to return the nonce in the user info request
```suggestion
      render json: AccountToUserInfo.new.(current_token.authorization.account, current_token.authorization.scopes)
```

Reply:
When integrating with ABM (apple business manage), nonce is required

Reply:
I can't find that in the spec. I'd prefer to remain spec compliant, but I would be willing to introduce some quirks provided they are documented